### ISSUE TITLE
Extend other modes of handling sensitive data to Flutter

### ISSUE THREAD

**Issue by andre-paraense** [smartlook/smartlook-mobile-issue-tracker#108]
**Platform**
Android, iOS and Web.

**Technology**
Flutter

**Summary**
Extend other modes of handling sensitive data to Flutter

Pragmatically, in order to be able to actually use Smartlook for Flutter in applications in which most of the screens handle PII data, we need the other modes of handling sensitive data, such as "Individual elements sensitivity", to be extended for this platform technology:

<img width="1282" alt="Screenshot 2021-10-07 at 11 17 05" src="https://user-images.githubusercontent.com/4104595/136366185-93f5d13f-7050-4170-ba43-f56128440480.png">

**Comment by pavel-kroh-smartlook** [comment]
Hi Andre, 

thanks for reaching us with the suggestion.

The good news is that we… are actually working on it :-) 

There is a soon to be released update that introduces the wireframe mode to Flutter, with more to come on this front.

Pavel

**Comment by SenNeonoveNoci** [comment]
@andre-paraense 
> Do you think that the wireframe rendering mode will be capable of automatically hiding sensitive data view components for apps written in Flutter?

Yeah, that's the main purpose of the `WIREFRAME` rendering mode. Hopefully, it's gonna suit your needs.

**Comment by andre-paraense** [comment]
I see that this has been closed.

Even though wireframe mode is useful, it is not ideal. Do you have any plans of supporting `individual elements sensitivity` and `classes sensitivity` for Flutter?
